packages/ui/src/theme/tokens/components/inAppMessaging/backdrop.ts

```diff
@@ -0,0 +1,9 @@
+import { BackgroundColorValue, DesignToken } from '../../types/designToken';
+
+export interface BackdropTokens {
```

Comment:
```suggestion
export interface InAppMessagingTokens {
```

See authenticator as example: https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/tokens/components/authenticator.ts